From 3357df9074949ac150838f2595b5f0551a00930c Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Wed, 14 Oct 2015 19:11:42 +0800
Subject: [PATCH 0592/5242] MLK-11711-01 ARM: dts:
 imx7d-12x12-lpddr3-arm2-sai: enable sai<->wm8958
 sound card support in lpsr mode

commit  2374dfdd3634706d4f1a90568ce73eb019e6f5a0 from
https://source.codeaurora.org/external/imx/linux-imx.git

enable sai<->wm8958 sound card support in lpsr mode

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts
index c2c0da5..719efca 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts
@@ -21,12 +21,15 @@
 };
 
 &iomuxc {
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_hog_1>;
+	pinctrl-1 = <&pinctrl_hog_1>;
 };
 
 &sai1 {
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_sai1>;
+	pinctrl-1 = <&pinctrl_sai1>;
 	status = "okay";
 };
 
-- 
1.7.9.5

